app/src/main/java/com/mapbox/maps/testapp/examples/OfflineActivity.kt

```diff
@@ -112,7 +112,6 @@ class OfflineActivity : AppCompatActivity() {
           }
         }
         binding.container.addView(mapView)
-        mapView?.onStart()
```

Comment:
We should keep this, as this method call is out of the lifecycle of Activity.

Reply:
I tried locally and seems it doesn't matter if we keep it or not.

Reply:
but for consistency I will keep it.